R/msgmerge.R

```diff
@@ -1,6 +1,15 @@
 # split off from tools::update_pkg_po() to only run the msgmerge & checkPoFile steps
+
+# https://www.gnu.org/software/gettext/manual/html_node/msgmerge-Invocation.html
 run_msgmerge = function(po_file, pot_file) {
-  if (system(sprintf("msgmerge --update %s %s", po_file, shQuote(pot_file))) != 0L) {
+  cmd <- paste("msgmerge",
+    "--update",
+    "--previous", # show previous match for fuzzy matches
```

Comment:
This makes fuzzy matches considerably more useful because it shows you the old message and the new message side-by-side